geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/delta/AddsDeltaInfo.java

```diff
@@ -24,14 +24,11 @@
 import java.util.List;
 
 import org.apache.geode.DataSerializer;
+import org.apache.geode.internal.InternalDataSerializer;
 
 public class AddsDeltaInfo implements DeltaInfo {
```

Comment:
Could this class and the Enum constant associated with it be renamed to "SAddDeltaInfo" and "SADD" respectively?

Reply:
It could be done but I don't think it should. The DeltaInfo name should describe what they contain. It is possible for the same DeltaInfo class to be used by multiple operations so even though, currently, this one is only used by SADD I think its current name is good: it is a DeltaInfo that contains one or more "add" operations. The ones with plural names can contain more than one item.